sunpy/coordinates/tests/test_transformations.py

```diff
@@ -388,6 +389,7 @@ def test_hgs_hcc_sunspice():
     assert_quantity_allclose(new.z, 688539.32*u.km, atol=1e-2*u.km)
 
 
+@pytest.mark.skipif(astropy.__version__ < '3.2.0', reason="Not supported by Astropy <3.2")
```

Comment:
In principle, you should use `pkg_resources.parse_version()` (or similar) for proper version comparison, although of course it's incredibly unlikely that this conditional will still be present in our code when there's Astropy 10.0 or if there's ever Astropy 3.10.

Reply:
Yeah I should.